builder/rootfs/bin/boot

```diff
@@ -119,6 +119,12 @@ function gen_host_keys {
 
 gen_host_keys
 
+# wait until confd generates the push-images script
+while [[ ! -e /usr/local/bin/push-images ]]; do
+	echo "waiting the generation of script '/usr/local/bin/push-images'..."
```

Comment:
let's change this to "waiting for confd to generate '/usr/local/bin/push-images'..."